dmpr.py

```diff
@@ -24,16 +28,41 @@
 """
 
 
-class ConfigurationException(Exception): pass
+@functools.lru_cache(maxsize=2**10)
+def normalize_network(network):
+    return str(ipaddress.ip_network(network, strict=False))
+
+
+def get_mutable_default(dictionary: dict, key, default_factory):
+    if key not in dictionary:
+        dictionary[key] = default_factory()
+    return dictionary[key]
+
 
+def dict_reverse_lookup(d: dict, value):
+    return list(d.keys())[list(d.values()).index(value)]
 
-class InternalException(Exception): pass
+
+class ConfigurationException(Exception):
+    pass
+
+
+class InternalException(Exception):
+    pass
 
 
 class DMPRConfigDefaults(object):
-    rtn_msg_interval = "30"
-    rtn_msg_interval_jitter = str(int(int(rtn_msg_interval) / 4))
-    rtn_msg_hold_time = str(int(rtn_msg_interval) * 3)
+    rtn_msg_interval = 30
+    rtn_msg_interval_jitter = rtn_msg_interval // 4
```

Comment:
The jitter should be in sub seconds resolution. Just to avoid so called global synchronization effects.